projects/angular/src/layout/_card.clarity.scss

```diff
@@ -169,27 +188,69 @@
     }
   }
 
-  .card-header,
-  .card-block,
+  .card-block {
+    @include mixins.css-var(
+      padding,
+      clr-card-block-padding,
+      card-variables.$card-children-padding-vertical card-variables.$card-children-padding-horizontal,
+      variables.$clr-use-custom-properties
+    );
+  }
+
   .card-footer {
-    padding: card-variables.$card-children-padding-vertical card-variables.$card-children-padding-horizontal;
+    @include mixins.css-var(
+      padding,
+      clr-card-footer-padding,
+      card-variables.$card-children-padding-vertical card-variables.$card-children-padding-horizontal,
+      variables.$clr-use-custom-properties
+    );
   }
 
-  .card-header,
+  h3.card-header,
+  .card-header {
```

Comment:
Wouldn't just `.card-header` do exactly the same thing?

Reply:
Same thing for `.card-title`.

Reply:
If it's with `.card-header` only and in a h3, we don't select it explicitly and it being overwritten due to the lower priority.

Reply:
Overwritten by the `h3` typography styles? Okay, that makes sense.